src/discof/exec/fd_exec_tile.c

```diff
@@ -248,19 +248,24 @@ hash_accounts( fd_exec_tile_ctx_t *                ctx,
 
   ulong             start_idx = msg->start_idx;
   ulong             end_idx   = msg->end_idx;
-  fd_lthash_value_t lt_hash;
-  fd_lthash_zero( &lt_hash );
 
   fd_accounts_hash_task_info_t * task_info = fd_wksp_laddr( ctx->runtime_public_wksp, msg->task_infos_gaddr );
   if( FD_UNLIKELY( !task_info ) ) {
     FD_LOG_ERR(( "Unable to join task info array" ));
   }
 
+  fd_lthash_value_t * lthash = fd_wksp_laddr( ctx->runtime_public_wksp, msg->lthash_gaddr );
+  if( FD_UNLIKELY( !lthash ) ) {
+    FD_LOG_ERR(( "Unable to join lthash" ));
+  }
```

Comment:
should just use `laddr_fast` and assert in `unprivileged_init` `ctx->runtime_public_wksp` is non-NULL and `msg->lthash_gaddr` is non-zero for a more useful error